data_model/src/permissions.rs

```diff
@@ -43,21 +81,29 @@ pub type Permissions = btree_set::BTreeSet<PermissionToken>;
 #[cfg_attr(feature = "ffi", derive(IntoFfi, TryFromFfi))]
 #[cfg_attr(feature = "ffi", ffi_export)]
 #[getset(get = "pub")]
-#[display(fmt = "{name}")]
+#[display(fmt = "{id}")]
 pub struct PermissionToken {
     /// Name of the permission rule given to account.
-    name: Name,
+    id: <Self as Identifiable>::Id,
     /// Params identifying how this rule applies.
     #[getset(skip)]
     params: btree_map::BTreeMap<Name, Value>,
 }
 
+impl Identifiable for PermissionToken {
```

Comment:
this warrants a discussion. `PermissionToken` wasn't `Identifiable` on purpose as far as I remember. Please contact @Arjentix for his work on #2421

Reply:
also, never implement `Identifiable` by hand without a specific reason now that we have the macro

Reply:
I think that you can have `PermissionToken` of the same name in two different accounts or two different roles. Maybe you can come up with an appropriate `Id` for `PermissionTokens`, but in any case it can't be identified only with name

Reply:
considering that we store permissions in both accounts and roles located in wsv, I can propose something like this for you:
```rs
    struct Id {
        name: Name,
        scope: Either<AccountId, RoleId>
    }
```

Reply:
`PermissionTokens` differ not only by their *ids* but also by their *parameter values*. `PermissionTokenDefinition` can be differed only by *id*, but it's not true for `PermissionToken` it-self. And I don't think we need this

Reply:
Right, that was a blunder.
I don't think I'll need it with `PermissionTokenDefinition` anyway.

Reply:
> `PermissionTokens` differ not only by their _ids_ but also by their _parameter values_. `PermissionTokenDefinition` can be differed only by _id_, but it's not true for `PermissionToken` it-self. And I don't think we need this

is this really the case? You're saying that account could have two permissions of the same name with different parameters?

Reply:
> `PermissionTokens` differ not only by their _ids_ but also by their _parameter values_. `PermissionTokenDefinition` can be differed only by _id_, but it's not true for `PermissionToken` it-self. And I don't think we need this

yeah, I was wrong there